cryptography/src/secp256r1/mod.rs

```diff
@@ -20,9 +20,9 @@
 //! let signature = signer.sign(namespace, msg);
 //!
 //! // Verify the signature
-//! assert!(Secp256r1::verify(namespace, msg, &signer.public_key(), &signature));
+//! assert!(signer.public_key().verify(namespace, msg, &signature));
```

Comment:
nit: I wonder if we can (should) define a default function on `PrivateKey` to `verify()`